integration

```diff
@@ -687,6 +687,7 @@
   "oziee/ha-solcast-solar",
   "pail23/stiebel_eltron_isg_component",
   "parautenbach/hass-shairport-sync",
+  "parvez/network_scanner",
```

Comment:
Are you still working on this?

Reply:
the PR is ready. could you please help review!